src/common/SwipeableFlatList/index.js

```diff
@@ -0,0 +1,199 @@
+// The component is temporary stolen from
```

Comment:
read this comment

Reply:
Nice -- it sounds like that code is crappy and I am fine maintaining our own version of it.